src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -38,7 +45,10 @@ public Long elementCount(List<String> elements, String element) {
      * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        return elements.stream()
+                .filter(el -> el.equals(element))
+                .findFirst()
+                .orElseThrow(NoSuchElementException::new);
```

Comment:
```suggestion
                .orElseThrow();
```
read about this method - which exception is thrown by default?

Reply:
Убрал